Source/Core/Core/HW/SI/SI_DeviceGBA.h

```diff
@@ -16,51 +16,51 @@
 
 namespace SerialInterface
 {
-u8 GetNumConnected();
-int GetTransferTime(u8 cmd);
 void GBAConnectionWaiter_Shutdown();
 
 class GBASockServer
 {
 public:
-  explicit GBASockServer(int device_number);
+  GBASockServer();
   ~GBASockServer();
 
-  void Disconnect();
-
+  bool Connect();
+  bool IsConnected();
   void ClockSync();
-
   void Send(const u8* si_buffer);
   int Receive(u8* si_buffer);
 
 private:
+  void Disconnect();
```

Comment:
I can live with that; its just that my brain sees various things as kinda symmetric - and this isn't symmetric so I put a comment here ;)